cmake/virtualenv.cmake

```diff
@@ -35,6 +35,10 @@ function(virtualenv_install)
       COMMAND ${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME} -m pip install --upgrade setuptools
       COMMAND_ECHO STDOUT
     )
+    execute_process(
+      COMMAND ${VIRTUALENV_BIN_DIR}/${VIRTUALENV_PYTHON_EXENAME} -m pip install --upgrade nanobind
```

Comment:
Nit for the future: should check error return on all of these and fail.

Reply:
Thanks. Since I'm not using virtualenv anymore, I should remove this.